mobile/native/src/ln_dlc/mod.rs

```diff
@@ -587,37 +591,48 @@ fn derive_trades_from_filled_orders() -> Result<Vec<WalletHistoryItem>> {
                         open_order.direction,
                     )?;
 
+                    let close_position_fee =
+                        order_matching_fee_taker(order.quantity, execution_price).to_sat();
+
                     // Closing a position is an inbound "payment", because the DLC channel is closed
                     // into the Lightning channel.
                     let flow = PaymentFlow::Inbound;
                     let amount_sats = (trader_margin as i64 + pnl) as u64;
 
                     trades.push(WalletHistoryItem {
                         flow,
-                        amount_sats,
+                        amount_sats: amount_sats - close_position_fee,
```

Comment:
🤔 I don't get why we have to subtract here.

Reply:
because the users payout is

`trader_margin + pnl - fee`

maybe it would have been clearer if I added it in line 600?